### PR TITLE
Make verbosity of symtab2gb configurable

### PR DESCRIPTION
**Pull request by tautschnig**
Do not produce debug output (as goto_convert might do, e.g., reporting on destructor code being produced) by default.

- [x] Each commit message has a non-empty body, explaining why the change was made.
- n/a Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [x] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- n/a White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: src/symtab2gb/symtab2gb_parse_options.h [hunk 1]
```diff
@@ -17,6 +17,7 @@ Author: Diffblue Ltd.
 
 #define SYMTAB2GB_OPTIONS                                                      \
   "(" SYMTAB2GB_OUT_FILE_OPT "):"                                              \
+  "(verbosity):"
```
Comment (thomasspriggs) [inline src/symtab2gb/symtab2gb_parse_options.h]: Nitpick - Missing trailing `\` which should be there for merge-conflict / diff-minimisation related reasons.

Comment (tautschnig) [reply src/symtab2gb/symtab2gb_parse_options.h]: The problem here is that clang-format removes that one, so one has to switch off clang-format for that bit of code, which has other downsides...

Comment (tautschnig) [reply src/symtab2gb/symtab2gb_parse_options.h]: (And I haven't found an option to teach clang-format otherwise.)

**Review by thomasspriggs — APPROVED** [review]

**Review by martin-cs — APPROVED** [review]

File: src/symtab2gb/symtab2gb_parse_options.h (hunk 1 above)
Comment (peterschrammel) [reply src/symtab2gb/symtab2gb_parse_options.h]: We turn off clang-format in the other *_parse_options.h

Comment (tautschnig) [reply src/symtab2gb/symtab2gb_parse_options.h]: That was actually also the case here, my bad. I've added the backslash.

**Review by peterschrammel — APPROVED** [review]

**Pull request merged**